bundles/org.eclipse.ui.workbench.texteditor/src/org/eclipse/ui/texteditor/FindReplaceDialog.java

```diff
@@ -1128,6 +1139,22 @@ private void updateFindHistory() {
 		}
 	}
 
+	/**
+	 * Called after executed find action to update the history.
+	 */
+	private void updateReplaceHistory() {
+		if (okToUse(fReplaceField)) {
+			fReplaceField.removeModifyListener(fReplaceModifyListener);
+
+			// XXX: Workaround for Combo bug on Linux (see bug 404202 and bug 410603)
+			if (Util.isLinux())
```

Comment:
```suggestion
			if (Util.isLinux()) {
```

Now is a good opportunity to introduce braces into this code!

Reply:
Yes, I have added them at this place and the other, similar code location in 69af3d0.